release

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env ruby
+require 'fog'
+require 'dotenv'
+require 'json'
+require 'digest'
+require 'git'
+
+Dotenv.load
+
+class FileManager
+  ALL_RELEASES = "releases/all.json"
+  LATEST_RELEASE = "releases/latest.json"
+
+  attr_reader :bucket
+
+  def initialize(bucket)
+    @bucket = bucket
+  end
+
+  def upload!(filename, content)
+    file = bucket.files.new(
+      key: filename,
+      body: content,
+      public: true
+    )
+    file.save
+  end
+
+  def url(filename)
+    get_file(filename).public_url
+  end
+
+  def fetch(filename, default: "")
+    bucket.files.head(filename) ? get_file(filename).body : default
+  end
+
+  def all_releases
+    ALL_RELEASES
+  end
+
+  def latest_release
+    LATEST_RELEASE
+  end
```

Comment:
Yeah I wasn't super sure what you meant by your other thing.

I was thinking of doing `storage_manager.all_releases` to get the file, and `storage_manager.all_releases=` to assign it, but that gets kinda weird and I wouldn't _really_ expect an assignment to also result in an external call.